src/lib/TermBuilder.ml

```diff
@@ -235,7 +236,7 @@ struct
         cof_split
          (el @@ ap fam_line [i; j])
          (boundary j)  (fun q -> ap bdry_line [i; j; q])
-         (eq i r)      (fun q -> ap bdy [j])
+         (eq i r)      (fun q -> sub_out @@ ap bdy [j])
       )
```

Comment:
for posterity / so i don't forget again: the reason these are needed is that the definition of a path expands to a cubical subtype, so to introduce on you must sub/in and to use one you must sub/out.